wordpress/wp-content/plugins/cds-base/classes/Modules/Wpml/Wpml.php

```diff
@@ -11,97 +11,96 @@ public static function register()
         Installer::register();
         Cleanup::register();
 
-        self::addTranslatedIDsToPages();
+        $instance = new self();
+
+        add_action('rest_api_init', [$instance, 'addTranslatedIDsToPages']);
     }
 
-    private static function getTranslatedPost($post_id, $lang)
+    public function getTranslatedPost($post_id, $lang)
     {
```

Comment:
This one can remain private since it's only called in another method, but it's not really that important.